### ISSUE TITLE
Is there any reason why no demo for activation maximization for resnet?

### ISSUE THREAD

**Issue by ahnchive** [raghakot/keras-vis#219]
- [ x] Check that you are up-to-date with the master branch of keras-vis. You can update with:
pip install git+git://github.com/raghakot/keras-vis.git --upgrade --no-deps

- [x ] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

Thank you for interesting work! I was wondering if there's any reason why no notebook demo for activation maximization for Resnet?  I want to try activation maximization with my custom model which is just typical CNN with batchnorm layers, but it's not working very well (the images extracted are just random noise, not visible pattern). I checked that this works perfectly fine with VGG16, and was wondering batchnorm is the issue? Anybody knows answer to this by any chance? Thank you for any help or idea in advance.
